src/types/account.ts

```diff
@@ -68,3 +68,10 @@ export type CreditData = {
   /** Maximum credit card limit. */
   creditLimit?: number
 }
+
+export type DeserializedAccount = Omit<Account, 'creditData'> & {
```

Comment:
looks like the usage of these `Deserialized__` types is internal (private), right? In that case we shouldn't publicly export these types, it enlarges the SDK API and can be confusing. We could for example move them to other file and keep them for only internal use.